examples/charts/horizon.py

```diff
@@ -25,7 +25,8 @@
 output_file("horizon.html")
 
 hp = Horizon(
-    xyvalues, index='Date',
+    xyvalues, x_names=['Date'],
```

Comment:
Yah, that's a fair point... I am not sure I follow the argument about why `x` and `y` are not reasonable. @pzwang can you elaborate?